Dockerfile

```diff
@@ -1,10 +1,8 @@
 FROM kalilinux/kali-linux-docker
 
-MAINTAINER @viyatb viyat.bhalodia@owasp.org, @alexandrasandulescu alecsandra.sandulescu@gmail.com
+MAINTAINER r3naissance
```

Comment:
Please don't change this here - I'll add your name to a contributors section in the README.